assets/src/edit-story/elements/shared/index.js

```diff
@@ -56,7 +57,9 @@ export const elementWithBackgroundColor = css`
 
 export const elementWithFont = css`
   white-space: pre-wrap;
-  font-family: ${({ font }) => font?.family};
+  font-family: ${({ font }) => generateFontFamily(font)};
+  overflow-wrap: break-word;
+  letter-spacing: normal;
```

Comment:
This is the ✨  line. One way to fix the global rule from [here](https://github.com/google/web-stories-wp/blob/1444bd2eb205ea9b6024a5788282464ba781538a/assets/src/edit-story/app/layout/index.js#L45).